README.md

```diff
@@ -93,6 +93,7 @@ Customize configurations are supported starting with version 1.0.2 of the Applic
     | Key                   | Value/Types | Default Value | Description                                                                                  |
     | --------------------- | ----------- | ------------- | -------------------------------------------------------------------------------------------- |
     | InitializationTimeout | TimeSpan    | 00:02:00      | Maximum time to wait for spinning up the container. Accepted format: [d.]hh:mm:ss[.fffffff]. |
+    | DisableCounters       | Boolean     | false         | Sets to true to avoid adding performance counter telemetry initializer.                      |

```

Comment:
Consider `DisablePerformanceCounters` for extra clarity

Reply:
Point taken. Updated.